homeassistant/components/aladdin_connect/sensor.py

```diff
@@ -89,14 +98,17 @@ def __init__(
         device: DoorDevice,
         description: AccSensorEntityDescription,
     ) -> None:
-        """Initialize a sensor for an Abode device."""
+        """Initialize a sensor for an Aladdin Connect device."""
         self._device_id = device["device_id"]
         self._number = device["door_number"]
         self._name = device["name"]
+        self._model = device["model"]
         self._acc = acc
         self.entity_description = description
         self._attr_unique_id = f"{self._device_id}-{self._number}-{description.key}"
         self._attr_has_entity_name = True
+        if self._model == "01" and description.key != "rssi":
```

Comment:
Maybe use positive matching here, in case of new attributes?
```suggestion
        if self._model == "01" and description.key in ("battery_level", "ble_strength"):
```

Or as alternative you could add a new boolean property to the description "enable_on_model_01".

Reply:
That makes sense - much more robust to adding sensors in the future that also should not be enabled.